### PR TITLE
Added Client Connector

### PR DESCRIPTION
**Pull request by MaxtorCoder**
And some other small stuffs, just ignore the ClientWhoRequest structure. 🔒

### CODE REVIEW

File: Source/NexusForever.ClientConnector/Program.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            uint dwCreationFlags,
+            IntPtr lpEnvironment,
+            string lpCurrentDirectory,
+            ref STARTUPINFO lpStartupInfo,
+            out PROCESS_INFORMATION lpProcessInformation
+        );
+
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Type in your host name : ");
```
Comment (lucas-jones) [inline Source/NexusForever.ClientConnector/Program.cs]: Maybe the arguments could be loaded from a text / `.json`. Which if it doesn't exist will be created with the default of `localhost`. 

This way users won't have to keep inputting the host name every time they want to play :)

File: Source/NexusForever.WorldServer/Network/Message/Model/ClientWhoRequest.cs [hunk 2]
```diff
@@ -6,8 +6,41 @@ namespace NexusForever.WorldServer.Network.Message.Model
     [Message(GameMessageOpcode.ClientWhoRequest, MessageDirection.Client)]
     public class ClientWhoRequest : IReadable
     {
+        #region off_DB3184
```
Comment (lucas-jones) [inline Source/NexusForever.WorldServer/Network/Message/Model/ClientWhoRequest.cs]: Accidental :P

**Review by lucas-jones — COMMENTED** [review]
I've added a few comments

File: Source/NexusForever.ClientConnector/Program.cs (hunk 1 above)
Comment (MaxtorCoder) [reply Source/NexusForever.ClientConnector/Program.cs]: Good plan, thanks :D

File: Source/NexusForever.WorldServer/Network/Message/Model/ClientWhoRequest.cs (hunk 2 above)
Comment (MaxtorCoder) [reply Source/NexusForever.WorldServer/Network/Message/Model/ClientWhoRequest.cs]: What do you mean? :P

**Pull request merged**